keyboards/for_science/keymaps/default/keymap.c

```diff
@@ -0,0 +1,119 @@
+ * |-Shift-+-------+-------+-------+-------+   |-------+-------+-------+-------+-------|
+ * |   Z   |   X   |   C   |   V   |   B   |   |   N   |   M   |   ,   |   .   |   /   |
+ * |  F6   |  F7   |  F8   |  F9   |  F10  |   | PgDn  | Left  | Down  | Right |       |
+ * `-Shift---Layer-+-------+-------+-------+   |-------+-------+-------+---------Layer-'
+ *                 | Ctrl  |  Gui  |  Alt  |   | Space | Layer | Shift |
+ *                 |       |       |       |   | Funct |       |       |
+ *                 `------------------------   `-Layer-----------------'
+ */
+
+[_BASE] = LAYOUT(
+    KC_TAB,  KC_ESC,  KC_LPRN, KC_LCBR, KC_LBRC,    KC_RBRC, KC_RCBR, KC_RPRN, KC_QUOT, KC_BSPC,
+    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,       KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+    SFT_A,   KC_S,    KC_D,    KC_F,    KC_G,       KC_H,    KC_J,    KC_K,    KC_L,    KC_ENT,
+    SFT_Z,   LAY_X,   KC_C,    KC_V,    KC_B,       KC_N,    KC_M,    KC_COMM, KC_DOT,  LAY_SLS,
+                      KC_LCTL, KC_LGUI, KC_LALT,    LAY_SPC, LAYER,   KC_RSFT
+),
+
+[_LAYER] = LAYOUT(
+    KC_TILD, KC_GRV,  KC_LABK, KC_UNDS, KC_MINS,    KC_PLUS, KC_EQL,  KC_RABK, KC_BSLS, KC_DEL,
+    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,       KC_6,    KC_7,    KC_8,    KC_9,    KC_0,
+    SFT_F1,  KC_F2,   KC_F3,   KC_F4,   KC_F5,      KC_PGUP, KC_HOME, KC_UP,   KC_END,  KC_SCLN,
+    SFT_F6,  LAY_F7,  KC_F8,   KC_F9,   KC_F10,     KC_PGDN, KC_LEFT, KC_DOWN, KC_RGHT, _______,
+                      _______, _______, _______,    FUNCT,   _______, _______
+),
+
+[_MOD_LAYER] = LAYOUT(
+    KC_TILD, _______, _______, _______, _______,    KC_Q,    _______, _______, _______, _______,
+    KC_TAB,  _______, _______, _______, _______,    _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______,
+                      _______, _______, _______,    _______, _______, _______
+),
+
+[_FUNCT] = LAYOUT(
+    RESET,   _______, _______, _______, LOCK,       MAC_LCK, _______, _______, _______, MAGIC_SWAP_LALT_LGUI,
+    _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______,
+                      _______, _______, _______,    _______, _______, _______
+),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    if (keyboard_report->mods & MODS_ANY) {
+        if (record->event.pressed) {
+            layer_on(_MOD_LAYER);
+        } else {
+            layer_off(_MOD_LAYER);
+        }
+    }
+
+    if (record->event.pressed) {
+        if (keycode == KC_BSPC && keyboard_report->mods & MODS_ALT) {
+            SEND_STRING(SS_TAP(X_DELETE));
```

Comment:
IIRC, this compiles smaller: 
```suggestion
           tap_code(KC_DEL);
```

If you're using longer strings, then the `SEND_STRING` function is better.